lib/ddtrace/contrib/action_cable/instrumentation.rb

```diff
@@ -0,0 +1,34 @@
+module Datadog
+  module Contrib
+    module ActionCable
+      module Instrumentation
+        # Instruments ActionCable::Connection to ensure the top-level
+        # rack request has the correct resource for this WebSockets connection.
+        #
+        # Some of our common assumptions don't hold under WebSockets, one example
+        # being the http status code, which is -1 for this request.
+        module ActionCableConnection
+          def on_open
+            Datadog.tracer.trace('action_cable.on_open') do |span|
+              begin
+                span.resource = "#{self.class}#on_open"
+                span.span_type = Datadog::Ext::AppTypes::WEB
+
+                span.set_tag('action_cable.action', 'on_open')
```

Comment:
These tag names should probably be constants in `Ext` as well; looks like you defined one for `action` but not `connection`?

Reply:
✅